.eslintrc.js

```diff
@@ -35,7 +35,15 @@ module.exports = {
     "func-names" : ["warn", "as-needed"],
     "no-underscore-dangle": ["error", {
       "allowAfterThis": true
-    }]
+    }],
+    "quotes" : ["error", "single", { "allowTemplateLiterals": true }],
+    "arrow-parens" : "off",
+    "arrow-body-style" : "off",
+    "func-names" : "off",
```

Comment:
Interesting ... I have never tried `.then(function someName() { /* ... */})'` before.

It seems to me that callbacks do not need access to `this`, so they can be written more easily as arrow functions:
```js
.then((data) => {
  // do something without declaring this
});
```

That way the function's scope is the parent's scope and it behaves like any other code block.